addons/survey/models/survey_question.py

```diff
@@ -178,6 +178,10 @@ def default_get(self, fields_list):
         ]"""
     )
 
+    # lives sessions - settings
+    survey_session_speed_rating = fields.Boolean(related="survey_id.session_speed_rating")
+    survey_session_speed_rating_time_limit = fields.Integer(related="survey_id.session_speed_rating_time_limit", string="General Time limit (seconds)")
```

Comment:
Let's put these below "session_available", which is also a related field based on session.

Reply:
Hi @awa-odoo :wave:

Change done.